package.json

```diff
@@ -165,25 +165,25 @@
     "@types/debug": "^4.1.5",
     "@types/global-modules": "^2.0.0",
     "@types/globjoin": "^0.1.0",
-    "@types/lodash": "^4.14.150",
+    "@types/lodash": "^4.14.151",
     "@types/micromatch": "^4.0.1",
     "benchmark": "^2.1.4",
     "common-tags": "^1.8.0",
     "del": "^5.1.0",
-    "eslint": "^6.8.0",
+    "eslint": "^7.0.0",
```

Comment:
[suggestion] How about adding `root:true`? Since I have my `~/.eslintrc.js` on my local machine, the following warning is shown:

```console
$ npm run lint:js --silent
(node:64924) [ESLINT_PERSONAL_CONFIG_SUPPRESS] DeprecationWarning: '~/.eslintrc.*' config files have been deprecated. Please remove it or add 'root:true' to the config files in your projects in order to avoid loading '~/.eslintrc.*' accidentally. (found in "../../../.eslintrc.js")
(Use `node --trace-deprecation ...` to show where the warning was created)
```

The following change can dismiss the warning:

```diff
diff --git a/package.json b/package.json
index a31681ff..c752e23a 100644
--- a/package.json
+++ b/package.json
@@ -58,6 +58,7 @@
   },
   "prettier": "@stylelint/prettier-config",
   "eslintConfig": {
+    "root": true,
     "extends": [
       "stylelint"
     ],
```